app/src/main/res/drawable/add_image.xml

```diff
@@ -0,0 +1,3 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android" android:width="24dp" android:height="24dp" android:viewportWidth="22" android:viewportHeight="22">
+    <path android:fillColor="#545454" android:pathData="M 3 3 L 3 4 L 3 19 L 4 19 L 15 19 L 12 16 L 15 13 L 17 13 L 16 12 L 14 10 L 10 14 L 9 13 L 4.1484375 17.851562 L 4 18 L 4 4 L 18 4 L 18 12 L 19 12 L 19 4 L 19 3 L 4 3 L 3 3 z M 7 5 C 5.892 5 5 5.892 5 7 C 5 8.108 5.892 9 7 9 C 8.108 9 9 8.108 9 7 C 9 5.892 8.108 5 7 5 z M 16 14 L 16 16 L 14 16 L 14 17 L 16 17 L 16 19 L 17 19 L 17 17 L 19 17 L 19 16 L 17 16 L 17 14 L 16 14 z "/>
+</vector>
```

Comment:
Can you please move icons you have from one source to a folder each.

- res/drawable/kde/breeze-icons
- res/drawable/simpleMobileTools/simple-gallery
- res/drawable/simpleMobileTools/simple-commons

Reply:
Just verified, SimpleMobileTools basically uses the Material Design Icons from Google. So I think there is no separation or additional credits required. And the only one form KDE Breeze is the placeholder image, which I have modified a little bit. Should it sit in a separate folder?

Reply:
Ok I found all the icons here as well: https://fonts.google.com/icons

In that case please just rename the kde icon to kde_breeze_icon_add_image just to be sure.

Reply:
Finally I created that placeholder icon form scratch on my own. It even looks slightly prettier now :D

Reply:
Awesome, looks great 🎉